src/petals/server/server.py

```diff
@@ -125,7 +131,7 @@ def __init__(
         self.dht = DHT(
             initial_peers=initial_peers,
             start=True,
-            num_workers=self.block_config.n_layer,
+            num_workers=self.block_config.num_hidden_layers,
```

Comment:
sanity check: is this field guaranteed for all models or only for bloom and llama? couldn't find it

Reply:
I think this field is standard because I saw models like BLOOM remapping model-specific config keys like `n_layer` to `num_hidden_layers` for compatibility (not vice versa): https://github.com/huggingface/transformers/blob/6ab045d6fe7a859ddc219cd144e638bb4d8ab2fe/src/transformers/models/bloom/configuration_bloom.py#L108